src/ConsoleConnector/Commands/Options/ParamName.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Autodesk.DataExchange.ConsoleApp.Commands.Options
+{
+    internal class ParamName:CommandOption
+    {
+        public new Autodesk.Parameters.Parameter? Value { get; set; }
+        public string ParameterName { get; set; }
+        public ParamName()
+        {
+            this.Description = "Specify the name for parameter";
+        }
+
+        public override void SetValue(string value)
+        {
+            Value = null;
+            ParameterName = value;
+            if (Enum.TryParse(value, true, out Autodesk.Parameters.Parameter parameter))
```

Comment:
if the tryParse fails we should throw an Invalid Args exception specifying the error and argument(nameof) that was invalid. The command invoker can catch the specific exception and return/show the error on the console. 
The "ParameterName = value;" lineshould come after the tryparse and execute only if we were able to parse the value properly.

Note; This kind of error handling and reporting should be there for all commands, if not there already. Commands should not fail silently. Can be a separate Jira.

Reply:
We will try to parse user's parameter to existing names. If matches, then we can add it as a built it parameter otherwise custom parameter. User can get list of build in parameter list with help command. If somehow user make typo mistake, then we are providing some information to the user then user can decide its correct or not.  If its not correct, then removeinstanceparam and removetypeparam commands are added. Jira [DEXC-1855](https://jira.autodesk.com/browse/DEXC-1855) is added to improve SDK.